src/pages/Pages/specials/Index/ChangelogCard.tsx

```diff
@@ -0,0 +1,59 @@
+import { Typography, styled } from '@mui/material';
+import Expand from 'components/layout/Expand';
+import Paper from 'components/layout/Paper';
+import MarkdownRenderer from 'components/miscellaneous/MarkdownRenderer';
+import { useQuery } from 'react-query';
+
+const LATEST_VERSION_INDEX = 3;
+const MARKDOWN_HEADER_DELIMITER = /(?=\n##\s *)/g;
+
+const ChangelogList = styled('div')({
+  '& ul': {
+    listStyleType: 'none',
+  },
+});
+
+const Expansion = styled(Expand)(({ theme }) => ({
+  border: `1px solid ${theme.palette.divider}`,
+}));
+
+export type WorkDoneCardProps = {
+  changelogURL: string;
+  title: string;
+};
+
+const getReleaseAsStringArray = async (changelogURL: string) => {
+  const res = await fetch(changelogURL);
+  const text = await res.text();
+  const markdownSections = text.split(MARKDOWN_HEADER_DELIMITER);
```

Comment:
Du kan returnere allerede på denne linjen
```suggestion
  return text.split(MARKDOWN_HEADER_DELIMITER);
```